### ISSUE TITLE
Can't sync if bookmark root is not English

### ISSUE THREAD

**Issue by goodideal** [Cleod9/syncmarx-webext#9]
Find in 0.6.5, two lines below restrict bookmark language. If bookmark root is not English, reports 'remoteFolder is null' error.

```
// BookmarkManager.js
var MENU_TITLES = /(^Bookmarks Menu$)|(^Other bookmarks$)|(^Bookmarks$)/gi;
var TOOLBAR_TITLES = /(^Bookmarks Toolbar$)|(^Bookmarks bar$)/gi;
```

**Comment by goodideal** [comment]
The temporary way is adding own language into regex.

**Comment by Cleod9** [comment]
Yes you are correct, my ideal strategy would actually be to remove this regex and allow the user to manually pick which bookmark folders to sync. So something like requiring the user to select a "Root Bookmarks" and "Bookmarks Bar" separatel (and maybe allowing others like Mobile Bookmarks). What's tricky though is that I'd like for the extension to be as easy to set up as possible, so I may have to make that an option for advanced users. I do like the idea of allowing the user to provide their own regex though, which might be simple enough in the meantime for most users who can just type a basic string.

Note that in 0.7.0 I made a change to perform a failover search if root bookmark folders can't be found by name which does fix the problem for some users:
https://github.com/Cleod9/syncmarx-webext/commit/1a3f3ab62eaeeba5fc75dda3cb80f68f253f8a04#diff-ca9eabbb818bb16073f78e0270adc259R289

But of course this fix isn't perfect since it seems those ids aren't necessarily guaranteed to be the same across different machines and browsers.

**Comment by goodideal** [comment]
I'm using floccus too. It allows user to select bookmark root for sync. I think most users would know what that means, and for better practicability. 

Any thoughts?

**Comment by Cleod9** [comment]
I think that's probably the way to go then, bookmark root it is. It will take some time for me to get to this since I've been pretty busy lately but it'll be on my to-odo list
